### PR TITLE
Add missing urls for AtomicPtr

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @frewsxcv

### CODE REVIEW

File: src/libcore/sync/atomic.rs [hunk 1]
```diff
@@ -773,11 +791,11 @@ impl<T> AtomicPtr<T> {
     /// return value is a result indicating whether the new value was written and containing the
     /// previous value.
     ///
-    /// `compare_exchange_weak` takes two `Ordering` arguments to describe the memory
+    /// `compare_exchange_weak` takes two [`Ordering`] arguments to describe the memory
     /// ordering of this operation. The first describes the required ordering if the operation
     /// succeeds while the second describes the required ordering when the operation fails. The
-    /// failure ordering can't be `Release` or `AcqRel` and must be equivalent or weaker than the
-    /// success ordering.
+    /// failure ordering can't be [`Release`] or [`AcqRel`] and must be equivalent or
```
Comment (frewsxcv) [inline src/libcore/sync/atomic.rs]: Where are these links linking to?

Comment (GuillaumeGomez) [inline src/libcore/sync/atomic.rs]: Oh damn.

**Comment by GuillaumeGomez** [comment]
Updated.

**Pull request merged**